### ISSUE TITLE
Maxmind geoip databases nolonger public

### ISSUE THREAD

**Issue by swalladge** [piqueserver/piqueserver#572]
Cf. https://twitter.com/maxmind/status/1207386233187516418

We will need to develop an alternate method to pulling in the geoip database in the future.

**Comment by NotAFile** [comment]
Oh, damn. That's super annoying.

**Comment by tomlister** [comment]
Does the license need to be compatible? Because the blog says the user just needs to agree to the license.

**Comment by swalladge** [comment]
> Does the license need to be compatible?

I don't think so, if the end user still downloads the file themselves (as is currently). But I'm not sure we'd be allowed to distribute it in the repo any more.

I think for now the easiest option would be to change the download link to a mirror of the latest database revision to be published under the creative commons license.

Then we can worry about the logistics of using the official one later.
